code/index.html

```diff
@@ -3,28 +3,210 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <title>Design & Development</title>
+    <link href="https://fonts.googleapis.com/css?family=Lora|Roboto:400,700" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!--Header with hero-image-->
+    <header class="hero-image">
+      <h1>Design & Development. </h1>
+      <h3>Help you create the nicest websites you have ever seen!</h3>
```

Comment:
Now this is nitpicking but try to keep the headings in numeric order, the `<h3>`should probably be replaced by a `<h2>`to make more sense, since it's styled bigger with the font-size. Something to keep in mind as you continue!